### PR TITLE
Fix the wrong import in setupTypeScript.js

### PR DESCRIPTION
**Pull request by Dexterp37**
Checklist for reviewer:

- [ ] The description should reference a bug or github issue, if relevant.
- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

File: scripts/setupTypeScript.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
 */
 
 import fs from "fs";
-import { dirname, path } from 'path';
+import { dirname }, path from 'path';
```
Comment (brizental) [inline scripts/setupTypeScript.js:20]: Hey, are you sure this works? I just tried it in Glean.js and I have to do:

```js
import path from "path";
import { dirname } from "path";
```

Even in Javascript files.

**Review by brizental — CHANGES_REQUESTED** [review]

File: scripts/setupTypeScript.js (hunk 1 above)
Comment (Dexterp37) [reply scripts/setupTypeScript.js:20]: Well, [yes](https://github.com/mozilla-rally/rally-core-addon/blob/a3052afad8dbd443b40c7edd9ea0da4a64179bdc/tests/core-addon/integration/utils.js#L10)?

Comment (brizental) [reply scripts/setupTypeScript.js:20]: Craziness. Why do I get an error...

**Review by brizental — APPROVED** [review]

File: scripts/setupTypeScript.js (hunk 1 above)
Comment (brizental) [reply scripts/setupTypeScript.js:20]: Ok, I have no documentation link, but I checked it out and it seems:
`import path, { dirname } from "path";` works,
`import { dirname }, path from "path";` doesn't.

Comment (Dexterp37) [inline scripts/setupTypeScript.js:20]:
```suggestion
import path, { dirname } from "path";
```

**Comment by Dexterp37** [comment]
Merging this, the CI changes are orthogonal and I'll be fixing them in anotehr PR.

**Pull request merged**